src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -13,7 +13,7 @@
     <!-- Declare a runtime dependency on the win8-arm CoreCLR built using the TFS builds -->
     <RuntimeDependency Include="runtime.win8-arm.Microsoft.NETCore.Runtime.CoreCLR">
       <TargetRuntime>win8-arm</TargetRuntime>
-      <Version>1.0.3-$(ExternalExpectedPrerelease)</Version>
+      <Version>1.0.4-$(ExternalExpectedPrerelease)</Version>
```

Comment:
Do we need to actually bump the arm version as it doesn't depend on the JIT package.